WIP 3.0 Chapters/Chapter 6 Spells.md

```diff
@@ -271,9 +271,14 @@
   
```

Comment:
One topic that needs to be had here, and going to post about it on the Discord too. 

We're basically settling for ourselves that the books we let ourselves reference are the core books. Tasha, Xanathar, and Mordekainen. 

But that would mean removing references to EGW, RotF, LLK, and SCAG fro the spells chapter. We should discuss that, and update this chapter accordingly.